quickjs-libc.c

```diff
@@ -190,6 +190,8 @@ static void js_set_thread_state(JSRuntime *rt, JSThreadState *ts)
     js_std_cmd(/*SetOpaque*/1, rt, ts);
 }
 
+#pragma GCC diagnostic push
```

Comment:
If we are going to work around the compiler, might as well be straight about it.